Pin News documentation V1 (#126)
exoplatform/exo-documentation: docs/GettingStarted.rst

|image167|

Pinning a News in the home page
-------------------------------

> If you have the membership "publisher" in the platform users group, you are allowed to pin articles in home page. This action is allowed from three locations.

Reviewer: ```suggestion
As a platform-wide publisher (publisher:/platform/users role required) If you have the membership "publisher", you are allowed to pin any articles to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```

1- Pinning a News from creation form : 

> From simple and complete creation forms, a checkbox "Pin article" is available. 

Reviewer: ```suggestion
From simple and full creation forms, a  "Pin article" checkbox is available. 
```

> As a platform-wide publisher (publisher:/platform/users role required) If you have the membership "publisher", you are allowed to pin any articles to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:

Reviewer: ```suggestion
As a platform-wide publisher (publisher:/platform/users role required), you are allowed to pin any article to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
```

|image168|

> When you type in fields with the article details, tick "Pin article" checkbox than click on "post" button a confirmation message appears.

Reviewer: ```suggestion
When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
```

|image169|

> By confirming the action the article is posted to space's activity stream and automatically pinned to home page News bloc. 

Reviewer: ```suggestion
After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc. 
```

> When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.

Reviewer: ```suggestion
After filling in the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
```

> 2- Pinning a News from activity stream : the "pin article" option is available from the three dots of the posted article activity.

Reviewer: ```suggestion
2- Pinning a News from activity stream : the "pin article" option is available from the three-dots menu of the article’s activity.
```

> After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc. 

Reviewer: ```suggestion
After confirmation, the article is posted to the space's activity stream and automatically published to the home page's News block. 
```

|image170|

> When you choose this option, a confirmation popup appears.

Reviewer: ```suggestion
When you choose this option, a confirmation popup appears:
```

|image169|

> A success message appears after confirming the action:

Reviewer: ```suggestion
After confirmation, a success message appears:
```

> 2- Pinning a News from activity stream : the "pin article" option is available from the three-dots menu of the article’s activity.

Reviewer: ```suggestion
2- Pinning a News from the activity stream : the "pin article" function is also available from the three-dots menu of the article’s activity.
```

> When you choose this option, a confirmation popup appears:

Reviewer: ```suggestion
When you select this entry, a confirmation popup appears:
```

Reviewer: @samehaidi  did you deliberately decide to not accept this suggestion or did you just miss it ?

Author: I decided to not accept this suggestion :)

Reviewer: Fine.

|image171|

*3- Pinning a News from News details*

> You may display the home page to verify that the news is available in the appropriate bloc.

Reviewer: ```suggestion
You may display the home page to verify that the news is available in the appropriate block.
```
